echopype/convert/set_groups_ek60.py

```diff
@@ -109,15 +108,8 @@ def __init__(self, *args, **kwargs):
 
     def set_provenance(self) -> xr.Dataset:
         """Set the Provenance group."""
-        # Collect variables
-        prov_dict = {
-            "conversion_software_name": "echopype",
-            "conversion_software_version": ECHOPYPE_VERSION,
-            "conversion_time": dt.utcnow().isoformat(timespec="seconds") + "Z",  # use UTC time
-            "src_filenames": self.input_file,
-            "duplicate_ping_times": 1 if self.old_ping_time is not None else 0,
-        }
-        # Save
+        prov_dict = echopype_prov_attrs(process_type="conversion", source_files=self.input_file)
+        prov_dict["duplicate_ping_times"] = 1 if self.old_ping_time is not None else 0
```

Comment:
Shall we change 1 to True and 0 to False to make it clear what this is for?
This is a temporary change for sure, just thought our intention to indicate that there was a change done here would be more clear:
```suggestion
        prov_dict["duplicate_ping_times"] = True if self.old_ping_time is not None else False
```

Reply:
geez, I didn't think about this, and it does not work!
https://github.com/pydata/xarray/issues/1500
I tested it out and it spilled the same error as stated in this open issue. Let's discard my suggestion!